package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "schulcloud-client",
-	"version": "25.3.0",
+	"version": "25.3.1",
```

Comment:
same version in the other hotfix

Reply:
I think that this PR will be merged first

Reply:
I'll be in contact with Lukasz

Reply:
thats fine, as the other hotfix is in the server

Reply:
keep in mind that https://github.com/hpi-schul-cloud/schulcloud-server/pull/2031#pullrequestreview-544953498 is on hte server, so the versions of those dont collide